source/about/on-prem-skype-for-business-replacement.rst

```diff
@@ -0,0 +1,51 @@
+On-Premises Skype for Business Replacement
+===========================================
+With Skype for Business reaching end-of-life, security-conscious organizations face a critical inflection point. Many operate in air-gapped or classified environments where cloud-based alternatives are not viable due to compliance restrictions, risk exposure, or data sovereignty mandates. Without a secure, modern, on-premises collaboration platform, these organizations risk operational disruption, mission misalignment, and non-compliance with stringent regulatory frameworks.
+
+Mattermost provides a secure, self-hosted communication and collaboration platform purpose-built for air-gapped environments, classified networks, and regulated industries. Designed to meet NIST 800-53, FedRAMP, and DISA STIG compliance requirements, Mattermost replaces legacy tools with modern capabilities—secure messaging, file sharing, workflow automation, and integrated video collaboration—while maintaining full enterprise control. Organizations can operate at scale, enable external collaboration without policy violations, and modernize their digital workflows without compromising security.
+
+The following mission-ready collaboration capabilities are available: 
+
+Air-Gapped and Classified Operations
+------------------------------------
+
+Organizations operating in fully disconnected or classified environments require secure communication platforms that function entirely within their own infrastructure.
+
+**Benefits**
+
+- **Ensure secure communication in fully disconnected networks** using Mattermost’s support for private on-premise deployments, including FIPS 140-2 validated and DISA STIG-hardened container images.
```

Comment:
Could link to deployment guide or [reference architecture overview](https://docs.mattermost.com/deploy/application-architecture.html)